src/RicardoFontanelli/LaravelTelegram/Telegram.php

```diff
@@ -222,10 +222,13 @@ public static function __callStatic($name, $arguments)
     {
         (!isset($this->resources[$name])) ? $this->resources[] = $name : null;
 
+        if (count($arguments) == 1 && is_array($arguments[0])) {
+            $arguments = $arguments[0];
```

Comment:
When entering `__call()` and `__callStatic()`, `$arguments` is an array of the given arguments.

So if you call, for instance, `forwardMessage($params)` with an array as the first parameter, `$arguments` will become an array containing `$params` in the first index. Which would be wrong to send to `callAPI()`